src/pages/Offer/sections/HedvigSwitch.tsx

```diff
@@ -1,21 +1,41 @@
 import { colors } from '@hedviginsurance/brand'
 import * as React from 'react'
 import styled from 'react-emotion'
-import { Card } from '.././components/Card'
-import { Header } from '.././components/Header'
-import { InnerWrapper } from '.././components/InnerWrapper'
-import { Wrapper } from '.././components/Wrapper'
+import { CardWrapper } from '../components/CardWrapper'
+import { HeaderWrapper } from '../components/HeaderWrapper'
+import { InnerWrapper } from '../components/InnerWrapper'
+import { Wrapper } from '../components/Wrapper'
 
 interface Props {
   title: string
 }
+const ROWWIDTH = 1200
```

Comment:
i think we should place these in a separate file and import them instead, that way we dont repeat ourselves

Reply:
I think most of them are local enough to keep them in file. Atm it would be us having a sizes file with only one export used in two places